### PR TITLE
Add query set and tests to Event Participant model

### LINKED ISSUE #62: Add query set and tests to Event Participants model
- Add query set to Event Participant model
- Add tests for the query set in the model

### PR DESCRIPTION
**Pull request by YaelD**
- Add query set and manager to Event Participant model
- Add query set tests

fix: #62

### CODE REVIEW

**Review by MortarDefender — DISMISSED** [review]
Great job,
Looks great 😁

**Comment by MortarDefender** [comment]
@YaelD
By the way, is it possible to change the name of participant_id to event_creator_id in the possible meeting class?

In my opinion the name participant_id can cause a bit of confusion with the other models who have the same name 

In addition i think the name possible meeting is a bit misleading,
Maybe the name possible meeting dates is more informative.

**Comment by YaelD** [comment]
> @YaelD By the way, is it possible to change the name of participant_id to event_creator_id in the possible meeting class?
> 

Yes, I will change it 😄 


> In my opinion the name participant_id can cause a bit of confusion with the other models who have the same name
> 
> In addition i think the name possible meeting is a bit misleading, Maybe the name possible meeting dates is more informative.

I agree that the name possible meeting is a bit misleading.
Maybe the name Optional Meetings or Optional Meeting Dates is more suitable?

**Comment by YaelD** [comment]
> @YaelD By the way, is it possible to change the name of participant_id to event_creator_id in the possible meeting class?
> 

Yes, I will change it 😄 


> In my opinion the name participant_id can cause a bit of confusion with the other models who have the same name
> 
> In addition i think the name possible meeting is a bit misleading, Maybe the name possible meeting dates is more informative.

I agree that the name possible meeting is a bit misleading.
Maybe the name Optional Meetings or Optional Meeting Dates is more suitable?

**Comment by MortarDefender** [comment]
> I agree that the name possible meeting is a bit misleading.
> Maybe the name Optional Meetings or Optional Meeting Dates is more suitable?

In my opinion the name "Optional Meeting Dates" is more suitable

**Review by MortarDefender — APPROVED** [review]
Looks Better with the new names 😄

**Review by MortarDefender — DISMISSED** [review]
Approved 😄

**Review by MortarDefender — DISMISSED** [review]

**Review by RonHachmon — DISMISSED** [review]
Great improvment

**Review by nemerna — DISMISSED** [review]

**Review by MortarDefender — DISMISSED** [review]

File: events/class_tests/participant_test.py [hunk 1]
```diff
@@ -48,3 +49,25 @@ def test_exist_event_participant(self):
...
             persist_event_participant.save()
+
+    def test_get_an_event_participants(self):
+        event = Event.objects.get(title="event1")
+        expected_participants = [
+            EventParticipant.objects.get(id=1),
+            EventParticipant.objects.get(id=2)
+        ]
+        all_participants_in_event1 = EventParticipant.objects.get_an_event_participants(event)
+        assert list(expected_participants) == list(all_participants_in_event1)
```
Comment (kobihk) [inline events/class_tests/participant_test.py]: expected_participants defined as list in line # 55 so I think you can do it like:
```suggestion
        assert expected_participants == list(all_participants_in_event1)
```

**Review by kobihk — CHANGES_REQUESTED** [review]

File: events/class_tests/participant_test.py (hunk 1 above)
Comment (YaelD) [reply events/class_tests/participant_test.py]: Done 😄

Comment (YaelD) [reply events/class_tests/participant_test.py]: Done 😄

**Review by kobihk — APPROVED** [review]

**Review by MortarDefender — APPROVED** [review]

**Review by RonHachmon — APPROVED** [review]

**Pull request merged**